firstvoices/backend/tests/test_models/test_alphabet_model.py

```diff
@@ -106,7 +106,6 @@ def test_clean_confusables_no_feeding(self):
         IgnoredCharacterFactory(site=alphabet.site, title="/")
         assert alphabet.clean_confusables("A/A") == "a/a"
 
-    @pytest.mark.skip("g2p with regex escape ignores mapping with escaped in-chars")
     @pytest.mark.django_db
     def test_clean_confusables_regex_escape(self):
```

Comment:
Would it work to re-query the alphabet in the test instead? Something like...

alphabet_created = AlphabetFactory.create(....)
alphabet = Alphabet.objects.only('input_to_canonical_map').get(id=alphabet_created.id)

assert ....